roles-in-tech/machine-learning.md

```diff
@@ -11,6 +11,7 @@ Example: 30-45 minutes
 ### Prerequisites
 
 Here are links to lessons that should be completed before this lesson:
+- [Data Science](https://github.com/Techtonica/curriculum/blob/master/roles-in-tech/data-science.md)
```

Comment:
Please use relative paths. Example, replace:
`https://github.com/Techtonica/curriculum/blob/master/html/html.md`
With:
`../html/html.md`

Reply:
This file is not on this branch so I am unable to edit it. Maybe it is removed by mistake. If it is necessary to do changes I put it here by copy-pasting from master.